backend-question-service/.gitignore

```diff
@@ -0,0 +1 @@
+.env
```

Comment:
would be nice to take a sample .gitignore for express projects somewhere else this will be filled with .vscode and other junk

Reply:
aight